spring-boot-project/spring-boot-docs/src/docs/asciidoc/anchor-rewrite.properties

```diff
@@ -663,6 +663,7 @@ howto-batch-applications=howto.batch
 howto-spring-batch-specifying-a-data-source=howto.batch.specifying-a-data-source
 howto-spring-batch-running-jobs-on-startup=howto.batch.running-jobs-on-startup
 howto-spring-batch-running-command-line=howto.batch.running-from-the-command-line
+howto-spring-batch-restarting-a-failed-job=howto.batch.restarting-a-failed-job
```

Comment:
I don't think this is necessary. anchor rewriting is only needed for changes to the doc where an old anchor has been renamed. As this is a new anchor, we don't need to update the `anchor-rewrite.properties`.

Reply:
OK, got that. I reverted the changes to  anchor-rewrite.properties.